metadata-ingestion/src/datahub/ingestion/source/lookml.py

```diff
@@ -478,6 +483,8 @@ def _get_upstream_lineage(self, looker_view: LookerView) -> UpstreamLineage:
 
     def _get_field_type(self, native_type: str) -> SchemaFieldDataType:
         field_type_mapping = {
+            **POSTGRES_TYPES_MAP,
```

Comment:
would be good to move this up to the top-level of the file (only initialize the dict once + allows dynamic modification if needed)